### PR TITLE
fix: remove vscode-commons dependency

### PR DESCRIPTION
**Pull request by AkshayBhansali18**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -284,12 +290,12 @@
     "webpack-cli": "^3.3.12"
   },
   "dependencies": {
-    "@redhat-developer/vscode-redhat-telemetry": "0.0.13",
+    "@redhat-developer/vscode-redhat-telemetry": "^0.2.0",
```
Comment (fbricon) [inline package.json:293]: 0.4.2

**Comment by fbricon** [comment]
you want to remove https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/blob/762d2e30bc0f7e7a884dd4dd52225590f7ee69a3/package.json#L9-L11

**Comment by mohitsuman** [comment]
@mathur07 Can you please review this PR and get it merged. We need to prioritise this.

**Comment by mohitsuman** [comment]
@AkshayBhansali18 Some tests are failing, can you please push a fix commit.

**Comment by mohitsuman** [comment]
@AkshayBhansali18 There is an error around the `record` entry for the import. Can you please fix that.

**Comment by mohitsuman** [comment]
@AkshayBhansali18 Thanks for fixing the tests. Though the `vsce package` is failing, please have a look and make sure the CI is green so we can proceed with merge.

**Pull request closed without merging**